toc.ts

```diff
@@ -98,7 +102,14 @@ export const TOC: TocGroup[] = [
       "streaming-files",
     ],
   },
-
+  {
+    title: "Databases",
```

Comment:
I assume this opens up the possibility for more database integrations. If this lands, would definitely want one for redis.